pkgs/applications/gis/grass/default.nix

```diff
@@ -37,6 +38,7 @@ stdenv.mkDerivation rec {
     "--with-readline"
     "--with-wxwidgets"
     "--with-netcdf"
+    "--with-pdal"
```

Comment:
On `darwin`, `pdal` should also similarly be disabled here pending a resolution to https://github.com/asmaloney/libE57Format/issues/48